Makefile

```diff
@@ -26,37 +29,17 @@ include $(FRAMEWORK_DIR)/moose.mk
 
 ALL_MODULES                 := no
 
-CHEMICAL_REACTIONS          := no
-CONTACT                     := no
-EXTERNAL_PETSC_SOLVER       := no
-FLUID_PROPERTIES            := no
-FUNCTIONAL_EXPANSION_TOOLS  := no
 HEAT_CONDUCTION             := yes
-LEVEL_SET                   := no
 MISC                        := yes
-NAVIER_STOKES               := no
-PHASE_FIELD                 := no
-POROUS_FLOW                 := no
-RDG                         := no
-RICHARDS                    := no
-SOLID_MECHANICS             := no
-STOCHASTIC_TOOLS            := no
 TENSOR_MECHANICS            := yes
-XFEM                        := no
```

Comment:
I think we could leave the extra modules in the makefile?  Someone might want to enable them for their own purposes.